ex2_hal/adcs/equipment_handler/src/adcs_handler.c

```diff
@@ -177,6 +182,162 @@ void get_3x3(float *matrix, uint8_t *address, float coef) {
     }
 }
 
+/*************************** File Management TC/TM Sequences ***************************/
+void ADCS_init_file_download_mutex() { adcs_file_download_mutex = xSemaphoreCreateMutex(); }
+
+ADCS_returnState ADCS_get_file_list() {
+    ADCS_returnState ret;
+
+    // Clear the file list
+    uint8_t index = adcs_file_list_length;
+    while (index--) {
+        vPortFree(adcs_file_list[index]);
+    }
+
+    // Fill the file list
+    ret = ADCS_reset_file_list_read_pointer();
+    if (ret != ADCS_OK) {
+        printf("Bad return at file list read pointer\n");
```

Comment:
this should be ex2_log. Probably a few of these in here, perhaps a find and replace